scene/theme/theme_owner.cpp

```diff
@@ -408,3 +363,19 @@ int ThemeOwner::get_theme_default_font_size() {
 	}
 	return ThemeDB::get_singleton()->get_fallback_font_size();
 }
+
+Ref<Theme> ThemeOwner::_get_owner_node_theme(Node *p_owner_node) const {
+	Ref<Theme> owner_theme;
+
+	Control *owner_c = Object::cast_to<Control>(p_owner_node);
+	if (owner_c) {
+		owner_theme = owner_c->get_theme();
```

Comment:
I think we can simplify this and return immediately in both cases. It was more awkward to write it like that when it was a part of a bigger method, but in a dedicated method we can remove unnecessary casts if we get a hit.

Then the last statement would just be `return Ref<Theme>();`.